mcs/class/System/Test/System.Diagnostics/PerformanceCounterTests.cs

```diff
@@ -0,0 +1,148 @@
+// originally copied from external/corefx
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Specialized;
+using NUnit.Framework;
+using System.Diagnostics;
+
+namespace MonoTests.System.Diagnostics
+{
+	internal class Common
```

Comment:
It doesn't matter that much for test code but this class isn't really how a C# developer would write it.

I'd restructure it to be a proper class that only exposes the needed properties:

```csharp
class PerformanceCounterHelper
{
    public string BaseName  { get; private set; }
    public string Name  { get; private set; }
    public string Category { get; private set; }
    public PerformanceCounter CounterSample { get; private set; }
    public PerformanceCounterCategory PerformanceCounterCategory { get; private set; }

    public PerformanceCounterHelper ()
    {
        var guid = Guid.NewGuid().ToString("N");
        Name = guid + "_Counter";
        Category = Name + "_Category";

        var counterDataCollection = new CounterCreationDataCollection();
        ...
     }
}
```